### PR TITLE
Upgraded to ember 1.12.0

### PR DESCRIPTION
**Pull request by Patsy-issa**
Upgraded ember list view to v 0.0.6
Added SafeString to style bindings
Part of #360

### CODE REVIEW

**Comment by rwjblue** [comment]
We chatted a bit about this in slack, and I think we decided that the extraction of `ListViewMixin` and reworking the various views that extend the internal `app/views/list.js` is not needed.

I think that @Patsy-issa is going to change that back (and confirm the tests still pass).

**Comment by Patsy-issa** [comment]
c changing back and adding comments to the files.

**Comment by rwjblue** [comment]
Looks good to me.

@teddyzeenny - r?

File: app/views/view-list.js [hunk 1]
```diff
@@ -1,27 +1,71 @@
...
+     * @method mouseEnter
+     * @param {Object} e event object
+     */
     mouseEnter(e) {
       this.get('controller').send('previewLayer', this.get('node'));
       e.stopPropagation();
     },
 
+    /**
+     * @property mouseLeave
```
Comment (teddyzeenny) [inline app/views/view-list.js]: s/property/method

**Comment by teddyzeenny** [comment]
Can you remove "Moved required properties into list-view mixin" from the commit message?

**Comment by Patsy-issa** [comment]
Updated commit message and added `SafeString` to style bindings.

**Comment by teddyzeenny** [comment]
:beers:

**Pull request merged**